### PR TITLE
doc: adding staking parameters explanation

### PR DESCRIPTION
**Pull request by kupermind**
- Adding staking parameters explanation.

### CODE REVIEW

**Review by DavidMinarsch — APPROVED** [review]

File: scripts/deployment/README.md [hunk 1]
```diff
@@ -25,6 +25,14 @@ Parameters of the `globals.json` file:
 - `useLedger`: a flag whether to use the hardware wallet (`true`) or proceed with the seed-phrase accounts (`false`);
 - `derivationPath`: a string with the derivation path;
 - `providerName`: a network type (see `hardhat.config.js` for the network configurations);
+- `serviceStakingParams`: a set of service staking parameters, which include:
+  - `rewardsPerSecond`: amount of token (in wei) per second credited to a service if the service is active enough. Assuming the maximum
+    number of reward per day is 1 (in ETH), then the reward per second is calculated by
```
Comment (mariapiamo) [inline scripts/deployment/README.md:30]:
```suggestion
    number of reward per week is 1 OLAS (in ETH), then the reward per second is calculated by
```

File: scripts/deployment/README.md [hunk 2]
```diff
@@ -25,6 +25,14 @@ Parameters of the `globals.json` file:
...
+    [this formula](https://www.wolframalpha.com/input?i=1+*+10%5E18+%2F+%283600+*+24+*+7%29);
+  - `livenessRatio`: number of service multisig transactions per second (in 1e18 value) that are used to measure the service
```
Comment (mariapiamo) [inline scripts/deployment/README.md:32]:
```suggestion
  - `livenessRatio`: number of service multisig transactions per second (with 18 decimals) that are used to measure the service
```

File: scripts/deployment/README.md [hunk 3]
```diff
@@ -25,6 +25,14 @@ Parameters of the `globals.json` file:
...
+    liveness (activity). In other words, it's the minimum number of transactions the service multisig needs to perform in order
+    to pass the liveness check. Assuming the number of required tx-s per day is 10, the liveness ratio is calculated by
+    [this formula](https://www.wolframalpha.com/input?i=%28115740740740740+*+60+*+60+*+24%29+%2F+10%5E18).
```
Comment (mariapiamo) [inline scripts/deployment/README.md:35]:
```suggestion
    [this formula](https://www.wolframalpha.com/input?i=%28115740740740740+*+86400%29%2F10%5E18).
```

File: scripts/deployment/README.md [hunk 4]
```diff
@@ -25,6 +25,14 @@ Parameters of the `globals.json` file:
 - `useLedger`: a flag whether to use the hardware wallet (`true`) or proceed with the seed-phrase accounts (`false`);
 - `derivationPath`: a string with the derivation path;
 - `providerName`: a network type (see `hardhat.config.js` for the network configurations);
+- `serviceStakingParams`: a set of service staking parameters, which include:
+  - `rewardsPerSecond`: amount of token (in wei) per second credited to a service if the service is active enough. Assuming the maximum
```
Comment (mariapiamo) [inline scripts/deployment/README.md:29]:
```suggestion
  - `rewardsPerSecond`: amount of token (in wei) per second credited to a service if the service is active enough.  The formula cn be obtained as follows`7days_rewards_per_service/7days_in_seconds` Assuming the maximum
```

File: scripts/deployment/README.md [hunk 5]
```diff
@@ -25,6 +25,14 @@ Parameters of the `globals.json` file:
...
+    number of reward per day is 1 (in ETH), then the reward per second is calculated by
+    [this formula](https://www.wolframalpha.com/input?i=1+*+10%5E18+%2F+%283600+*+24+*+7%29);
+  - `livenessRatio`: number of service multisig transactions per second (in 1e18 value) that are used to measure the service
+    liveness (activity). In other words, it's the minimum number of transactions the service multisig needs to perform in order
+    to pass the liveness check. Assuming the number of required tx-s per day is 10, the liveness ratio is calculated by
```
Comment (mariapiamo) [inline scripts/deployment/README.md:34]:
```suggestion
    to pass the liveness check. To check this `rewardsPerSecond* livenessPeriod/1e18` should approximate the number of txs required per livenessPeriod. Assuming the number of required tx-s per day is 10, the liveness ratio can be checked by means of
```

File: scripts/deployment/README.md (hunk 3 above)
Comment (kupermind) [reply scripts/deployment/README.md:35]: I think the previous one was better for understanding - seconds, minutes, hours

File: scripts/deployment/README.md (hunk 4 above)
Comment (kupermind) [reply scripts/deployment/README.md:29]: There's some issues with formatting, please check for spaces and names, as I am not sure how to correct that.

File: scripts/deployment/README.md (hunk 3 above)
Comment (mariapiamo) [reply scripts/deployment/README.md:35]: ok, then let's keep your :-)

File: scripts/deployment/README.md (hunk 4 above)
Comment (mariapiamo) [reply scripts/deployment/README.md:29]: let's ignore this. Later we can add more details if more info are required

**Pull request merged**